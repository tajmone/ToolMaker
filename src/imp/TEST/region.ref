hejsan%%(R1)happ
hejsan1hopp
hejsan1hipp
hejsanhupp
hejsanhepp
%%LOOP(L1,R2)
Parameter nr 1 = %%(R2[1])
%%ENDLOOP(L1)
Parameter nr 1 = A
Parameter nr 1 = A
Parameter nr 2 = B
Parameter nr 3 = 12
Parameter nr 1 = A
Parameter nr 2 = B
Parameter nr 3 = 12



%%INCLUDE(%%(G)PELLE (2,%%(VAR),%%(5+h*9)))
%%INCLUDE(PELLE (2,VAR,%%(5+h*9)))


  B = 13
  Hej svejs



Mary had a little lamb.
Peter had a little cat.
Mary had a little lamb
Peter had a little cat




1
1
1,2
A
1
{1,2}
1
2
{1,2}
{3,4}
LOCAL1                    V{rde p} P1 innan SET-satser
LOCAL2                    V{rde p} P2 innan SET-satser
LOCAL3                    V{rde p} P3 innan SET-satser
LOCAL4                    V{rde p} P4 innan SET-satser
                V{rde p} LOCAL1 innan SET-satser
                V{rde p} LOCAL2 innan SET-satser
                V{rde p} LOCAL3 innan SET-satser
                V{rde p} LOCAL4 innan SET-satser
LOCAL1                    V{rde p} P1 efter SET-satser
LOCAL2                    V{rde p} P2 efter SET-satser
LOCAL3                    V{rde p} P3 efter SET-satser
LOCAL4                    V{rde p} P4 efter SET-satser
Local1                V{rde p} LOCAL1 efter SET-satser
Local2                V{rde p} LOCAL2 efter SET-satser
Local3                V{rde p} LOCAL3 efter SET-satser
Local4                V{rde p} LOCAL4 efter SET-satser
%%(P1)                    V{rde p} P1 efter INCLUDE-satsen
%%(P2)                    V{rde p} P2 efter INCLUDE-satsen
%%(P3)                    V{rde p} P3 efter INCLUDE-satsen
%%(P4)                    V{rde p} P4 efter INCLUDE-satsen
Global1                V{rde p} LOCAL1 efter INCLUDE-satsen
Global2                V{rde p} LOCAL2 efter INCLUDE-satsen
Global3                V{rde p} LOCAL3 efter INCLUDE-satsen
Global4                V{rde p} LOCAL4 efter INCLUDE-satsen
Global1                    V{rde p} P1 innan SET-satser
Global2                    V{rde p} P2 innan SET-satser
Global3                    V{rde p} P3 innan SET-satser
Global4                    V{rde p} P4 innan SET-satser
                V{rde p} LOCAL1 innan SET-satser
                V{rde p} LOCAL2 innan SET-satser
                V{rde p} LOCAL3 innan SET-satser
                V{rde p} LOCAL4 innan SET-satser
Global1                    V{rde p} P1 efter SET-satser
Global2                    V{rde p} P2 efter SET-satser
Global3                    V{rde p} P3 efter SET-satser
Global4                    V{rde p} P4 efter SET-satser
Local1                V{rde p} LOCAL1 efter SET-satser
Local2                V{rde p} LOCAL2 efter SET-satser
Local3                V{rde p} LOCAL3 efter SET-satser
Local4                V{rde p} LOCAL4 efter SET-satser
%%(P1)                    V{rde p} P1 efter EINCLUDE-satsen
%%(P2)                    V{rde p} P2 efter EINCLUDE-satsen
%%(P3)                    V{rde p} P3 efter EINCLUDE-satsen
%%(P4)                    V{rde p} P4 efter EINCLUDE-satsen
Global1                V{rde p} LOCAL1 efter EINCLUDE-satsen
Global2                V{rde p} LOCAL2 efter EINCLUDE-satsen
Global3                V{rde p} LOCAL3 efter EINCLUDE-satsen
Global4                V{rde p} LOCAL4 efter EINCLUDE-satsen
Begin P1 = 0
R1[1] = 0

Local1 = 
Local1 = 0

Including with P1 = 1
 Begin P1 = 1
 R1[1] = 1

 Local1 = 
 Local1 = 1

 Including with P1 = 2
  Begin P1 = 2
  R1[1] = 2

  Local1 = 
  Local1 = 2

  Including with P1 = 3
   Begin P1 = 3
   R1[1] = 3

   Local1 = 
   Local1 = 3

   Including with P1 = 4
    Begin P1 = 4
    R1[1] = 4

    Local1 = 
    Local1 = 4

    Including with P1 = 5
     Begin P1 = 5
     R1[1] = 5

     Local1 = 
     Local1 = 5

     Including with P1 = 6
      Begin P1 = 6
      R1[1] = 6

      Local1 = 
      Local1 = 6

      Including with P1 = 7
       Begin P1 = 7
       R1[1] = 7

       Local1 = 
       Local1 = 7

       Including with P1 = 8
        Begin P1 = 8
        R1[1] = 8

        Local1 = 
        Local1 = 8

        Including with P1 = 9
         Begin P1 = 9
         R1[1] = 9

         Local1 = 
         Local1 = 9

         Including with P1 = 10
          Begin P1 = 10
          R1[1] = 10

          Local1 = 
          Local1 = 10


          Local1 = 10
          Local1 = 10

          R1[1] = 10
          End P1 = 10

         Local1 = 9
         Local1 = 9

         R1[1] = 10
         End P1 = 9

        Local1 = 8
        Local1 = 8

        R1[1] = 10
        End P1 = 8

       Local1 = 7
       Local1 = 7

       R1[1] = 10
       End P1 = 7

      Local1 = 6
      Local1 = 6

      R1[1] = 10
      End P1 = 6

     Local1 = 5
     Local1 = 5

     R1[1] = 10
     End P1 = 5

    Local1 = 4
    Local1 = 4

    R1[1] = 10
    End P1 = 4

   Local1 = 3
   Local1 = 3

   R1[1] = 10
   End P1 = 3

  Local1 = 2
  Local1 = 2

  R1[1] = 10
  End P1 = 2

 Local1 = 1
 Local1 = 1

 R1[1] = 10
 End P1 = 1

Local1 = 0
Local1 = 0

R1[1] = 10
End P1 = 0
