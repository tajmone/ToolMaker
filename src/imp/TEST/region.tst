%%BEGIN (R1)
hejsan%%(R1)%%END (R1)
%%INCLUDE (R1)happ
%%INCLUDE (R1 (1))hopp
%%INCLUDE (R1)hipp
%%INCLUDE (R1 ())hupp
%%INCLUDE (R1)hepp
%%-- Kolla elementen i parameter
%%BEGIN (R2)
%%SET I (1)
%%LOOP (L1, R2)
Parameter nr %%(I) = %%(R2[I])
%%ESET I (I+1)
%%ENDLOOP (L1)
%%END (R2)
%%INCLUDE (R2)
%%INCLUDE (R2 ())
%%INCLUDE (R2 (A))
%%INCLUDE (R2 (A,B,%%(3*4)))
%%INCLUDE (R2)



%%EINCLUDE (%%(G)PELLE    (2,VAR,15-10+h*9))
%%INCLUDE (PELLE    (2,VAR,%%(15-10+h*9)))

%%BEGIN (A (B))
  B = %%(B)
  Hej%%END (A)

%%INCLUDE (A (13)) svejs

%%BEGIN(aStory)
%%(aStory[1]) had a little %%(aStory[2]).
%%END (aStory)

%%BEGIN(bStory (p1, p2))
%%(p1) had a little %%(p2)
%%END (bStory)

%%INCLUDE(aStory(Mary, lamb, hund))
%%INCLUDE(aStory(Peter, cat, hund))
%%INCLUDE(bStory(Mary, lamb))
%%INCLUDE(bStory(Peter, cat))


%%SET A (1,2)

%%BEGIN (R99)
%%(%%(R99[2]))
%%(R99[2])
%%END(R99)

%%EINCLUDE(R99 (0,A,2))
%%INCLUDE(R99 (0,A,2))
%%BEGIN (R21 (P1))
%%(P1)
%%END (R21)
%%INCLUDE (R21 ())
%%INCLUDE (R21 (1))
%%INCLUDE (R21 (1,2))
%%INCLUDE (R21 ({1,2}))
%%BEGIN (R22 (P1, P2))
%%(P1)
%%(P2)
%%END (R22)
%%INCLUDE (R22 ())
%%INCLUDE (R22 (1))
%%INCLUDE (R22 (1,2))
%%INCLUDE (R22 ({1,2}, {3,4}))
%%INCLUDE (R22 (1,2,3,4))
%%SET LOCAL1 ("Global1")
%%SET LOCAL2 ("Global2")
%%SET LOCAL3 ("Global3")
%%SET LOCAL4 ("Global4")
%%BEGIN (R23 (P1, P2, P3, P4), LOCAL1, LOCAL2, LOCAL3, LOCAL4)
%%(P1)                    V{rde p} P1 innan SET-satser
%%(P2)                    V{rde p} P2 innan SET-satser
%%(P3)                    V{rde p} P3 innan SET-satser
%%(P4)                    V{rde p} P4 innan SET-satser
%%(LOCAL1)                V{rde p} LOCAL1 innan SET-satser
%%(LOCAL2)                V{rde p} LOCAL2 innan SET-satser
%%(LOCAL3)                V{rde p} LOCAL3 innan SET-satser
%%(LOCAL4)                V{rde p} LOCAL4 innan SET-satser
%%SET LOCAL1 ("Local1")
%%SET LOCAL2 ("Local2")
%%SET LOCAL3 ("Local3")
%%SET LOCAL4 ("Local4")
%%(P1)                    V{rde p} P1 efter SET-satser
%%(P2)                    V{rde p} P2 efter SET-satser
%%(P3)                    V{rde p} P3 efter SET-satser
%%(P4)                    V{rde p} P4 efter SET-satser
%%(LOCAL1)                V{rde p} LOCAL1 efter SET-satser
%%(LOCAL2)                V{rde p} LOCAL2 efter SET-satser
%%(LOCAL3)                V{rde p} LOCAL3 efter SET-satser
%%(LOCAL4)                V{rde p} LOCAL4 efter SET-satser
%%END (R23)
%%INCLUDE (R23 (LOCAL1, LOCAL2, LOCAL3, LOCAL4))
%%(P1)                    V{rde p} P1 efter INCLUDE-satsen
%%(P2)                    V{rde p} P2 efter INCLUDE-satsen
%%(P3)                    V{rde p} P3 efter INCLUDE-satsen
%%(P4)                    V{rde p} P4 efter INCLUDE-satsen
%%(LOCAL1)                V{rde p} LOCAL1 efter INCLUDE-satsen
%%(LOCAL2)                V{rde p} LOCAL2 efter INCLUDE-satsen
%%(LOCAL3)                V{rde p} LOCAL3 efter INCLUDE-satsen
%%(LOCAL4)                V{rde p} LOCAL4 efter INCLUDE-satsen
%%EINCLUDE (R23 (LOCAL1, LOCAL2, LOCAL3, LOCAL4))
%%(P1)                    V{rde p} P1 efter EINCLUDE-satsen
%%(P2)                    V{rde p} P2 efter EINCLUDE-satsen
%%(P3)                    V{rde p} P3 efter EINCLUDE-satsen
%%(P4)                    V{rde p} P4 efter EINCLUDE-satsen
%%(LOCAL1)                V{rde p} LOCAL1 efter EINCLUDE-satsen
%%(LOCAL2)                V{rde p} LOCAL2 efter EINCLUDE-satsen
%%(LOCAL3)                V{rde p} LOCAL3 efter EINCLUDE-satsen
%%(LOCAL4)                V{rde p} LOCAL4 efter EINCLUDE-satsen
%%BEGIN (R1 (P1, P2), Local1, Local2)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Begin P1 = %%(P1)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
R1[1] = %%(R1[1])

%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)
%%ESET Local1 (P1)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)

%%IF (P1 # P2)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Including with P1 = %%(P1+1)
%%EINCLUDE (R1 (P1+1, 10))
%%ENDIF

%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)
%%ESET Local1 (P1)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)

%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
R1[1] = %%(R1[1])
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
End P1 = %%(P1)
%%END (R1)
%%EINCLUDE(R1 (0, 10))
