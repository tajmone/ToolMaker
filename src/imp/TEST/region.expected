########## region ##########
%%BEGIN (R1)
hejsan%%(R1)%%END (R1)
%%INCLUDE (R1)happ
%%INCLUDE (R1 (1))hopp
%%INCLUDE (R1)hipp
%%INCLUDE (R1 ())hupp
%%INCLUDE (R1)hepp
%%-- Kolla elementen i parameter
%%BEGIN (R2)
%%SET I (1)
%%LOOP (L1, R2)
Parameter nr %%(I) = %%(R2[I])
%%ESET I (I+1)
%%ENDLOOP (L1)
%%END (R2)
%%INCLUDE (R2)
%%INCLUDE (R2 ())
%%INCLUDE (R2 (A))
%%INCLUDE (R2 (A,B,%%(3*4)))
%%INCLUDE (R2)



%%EINCLUDE (%%(G)PELLE    (2,VAR,15-10+h*9))
%%INCLUDE (PELLE    (2,VAR,%%(15-10+h*9)))

%%BEGIN (A (B))
  B = %%(B)
  Hej%%END (A)

%%INCLUDE (A (13)) svejs

%%BEGIN(aStory)
%%(aStory[1]) had a little %%(aStory[2]).
%%END (aStory)

%%BEGIN(bStory (p1, p2))
%%(p1) had a little %%(p2)
%%END (bStory)

%%INCLUDE(aStory(Mary, lamb, hund))
%%INCLUDE(aStory(Peter, cat, hund))
%%INCLUDE(bStory(Mary, lamb))
%%INCLUDE(bStory(Peter, cat))


%%SET A (1,2)

%%BEGIN (R99)
%%(%%(R99[2]))
%%(R99[2])
%%END(R99)

%%EINCLUDE(R99 (0,A,2))
%%INCLUDE(R99 (0,A,2))
%%BEGIN (R21 (P1))
%%(P1)
%%END (R21)
%%INCLUDE (R21 ())
%%INCLUDE (R21 (1))
%%INCLUDE (R21 (1,2))
%%INCLUDE (R21 ({1,2}))
%%BEGIN (R22 (P1, P2))
%%(P1)
%%(P2)
%%END (R22)
%%INCLUDE (R22 ())
%%INCLUDE (R22 (1))
%%INCLUDE (R22 (1,2))
%%INCLUDE (R22 ({1,2}, {3,4}))
%%INCLUDE (R22 (1,2,3,4))
%%SET LOCAL1 ("Global1")
%%SET LOCAL2 ("Global2")
%%SET LOCAL3 ("Global3")
%%SET LOCAL4 ("Global4")
%%BEGIN (R23 (P1, P2, P3, P4), LOCAL1, LOCAL2, LOCAL3, LOCAL4)
%%(P1)                    V{rde p} P1 innan SET-satser
%%(P2)                    V{rde p} P2 innan SET-satser
%%(P3)                    V{rde p} P3 innan SET-satser
%%(P4)                    V{rde p} P4 innan SET-satser
%%(LOCAL1)                V{rde p} LOCAL1 innan SET-satser
%%(LOCAL2)                V{rde p} LOCAL2 innan SET-satser
%%(LOCAL3)                V{rde p} LOCAL3 innan SET-satser
%%(LOCAL4)                V{rde p} LOCAL4 innan SET-satser
%%SET LOCAL1 ("Local1")
%%SET LOCAL2 ("Local2")
%%SET LOCAL3 ("Local3")
%%SET LOCAL4 ("Local4")
%%(P1)                    V{rde p} P1 efter SET-satser
%%(P2)                    V{rde p} P2 efter SET-satser
%%(P3)                    V{rde p} P3 efter SET-satser
%%(P4)                    V{rde p} P4 efter SET-satser
%%(LOCAL1)                V{rde p} LOCAL1 efter SET-satser
%%(LOCAL2)                V{rde p} LOCAL2 efter SET-satser
%%(LOCAL3)                V{rde p} LOCAL3 efter SET-satser
%%(LOCAL4)                V{rde p} LOCAL4 efter SET-satser
%%END (R23)
%%INCLUDE (R23 (LOCAL1, LOCAL2, LOCAL3, LOCAL4))
%%(P1)                    V{rde p} P1 efter INCLUDE-satsen
%%(P2)                    V{rde p} P2 efter INCLUDE-satsen
%%(P3)                    V{rde p} P3 efter INCLUDE-satsen
%%(P4)                    V{rde p} P4 efter INCLUDE-satsen
%%(LOCAL1)                V{rde p} LOCAL1 efter INCLUDE-satsen
%%(LOCAL2)                V{rde p} LOCAL2 efter INCLUDE-satsen
%%(LOCAL3)                V{rde p} LOCAL3 efter INCLUDE-satsen
%%(LOCAL4)                V{rde p} LOCAL4 efter INCLUDE-satsen
%%EINCLUDE (R23 (LOCAL1, LOCAL2, LOCAL3, LOCAL4))
%%(P1)                    V{rde p} P1 efter EINCLUDE-satsen
%%(P2)                    V{rde p} P2 efter EINCLUDE-satsen
%%(P3)                    V{rde p} P3 efter EINCLUDE-satsen
%%(P4)                    V{rde p} P4 efter EINCLUDE-satsen
%%(LOCAL1)                V{rde p} LOCAL1 efter EINCLUDE-satsen
%%(LOCAL2)                V{rde p} LOCAL2 efter EINCLUDE-satsen
%%(LOCAL3)                V{rde p} LOCAL3 efter EINCLUDE-satsen
%%(LOCAL4)                V{rde p} LOCAL4 efter EINCLUDE-satsen
%%BEGIN (R1 (P1, P2), Local1, Local2)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Begin P1 = %%(P1)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
R1[1] = %%(R1[1])

%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)
%%ESET Local1 (P1)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)

%%IF (P1 # P2)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Including with P1 = %%(P1+1)
%%EINCLUDE (R1 (P1+1, 10))
%%ENDIF

%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)
%%ESET Local1 (P1)
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
Local1 = %%(Local1)

%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
R1[1] = %%(R1[1])
%%LOOP (L1 <= P1) 
 %%--
%%ENDLOOP(L1)
End P1 = %%(P1)
%%END (R1)
%%EINCLUDE(R1 (0, 10))
########## ../imp region.tst >region.tst.tmp; cat region.tst.tmp; rm region.tst.tmp ##########


region.tst

    2.  hejsan%%(R1)%%END (R1)
=====>           1

  *1*   204 W : Variable "R1" not set.

   11.  %%LOOP (L1, R2)
=====>              1

  *1*   204 W : Variable "R2" not set.

   12.  Parameter nr %%(I) = %%(R2[I])
=====>                          1

  *1*   204 W : Variable "R2" not set.

   24.  %%EINCLUDE (%%(G)PELLE    (2,VAR,15-10+h*9))
=====>                 1             2         3

  *1*   204 W : Variable "G" not set.
  *2*   204 W : Variable "VAR" not set.
  *3*   204 W : Variable "h" not set.

   25.  %%INCLUDE (PELLE    (2,VAR,%%(15-10+h*9)))
=====>             1                        2

  *1*   205 W : Label "PELLE" not defined.
  *2*   204 W : Variable "h" not set.

   59.  %%INCLUDE (R21 ())
=====>  1

  *1*   213 E : Incorrect number of parameters given in INCLUDE statement

   61.  %%INCLUDE (R21 (1,2))
=====>  1

  *1*   213 E : Incorrect number of parameters given in INCLUDE statement

   67.  %%INCLUDE (R22 ())
=====>  1

  *1*   213 E : Incorrect number of parameters given in INCLUDE statement

   68.  %%INCLUDE (R22 (1))
=====>  1

  *1*   213 E : Incorrect number of parameters given in INCLUDE statement

   71.  %%INCLUDE (R22 (1,2,3,4))
=====>  1

  *1*   213 E : Incorrect number of parameters given in INCLUDE statement

   99.  %%(P1)                    V{rde p} P1 efter INCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P1" not set.

  100.  %%(P2)                    V{rde p} P2 efter INCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P2" not set.

  101.  %%(P3)                    V{rde p} P3 efter INCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P3" not set.

  102.  %%(P4)                    V{rde p} P4 efter INCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P4" not set.

  108.  %%(P1)                    V{rde p} P1 efter EINCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P1" not set.

  109.  %%(P2)                    V{rde p} P2 efter EINCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P2" not set.

  110.  %%(P3)                    V{rde p} P3 efter EINCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P3" not set.

  111.  %%(P4)                    V{rde p} P4 efter EINCLUDE-satsen
=====>     1

  *1*   204 W : Variable "P4" not set.


5 error(s) detected.
16 warning(s) issued.

hejsan%%(R1)happ
hejsan1hopp
hejsan1hipp
hejsanhupp
hejsanhepp
%%LOOP(L1,R2)
Parameter nr 1 = %%(R2[1])
%%ENDLOOP(L1)
Parameter nr 1 = A
Parameter nr 1 = A
Parameter nr 2 = B
Parameter nr 3 = 12
Parameter nr 1 = A
Parameter nr 2 = B
Parameter nr 3 = 12



%%INCLUDE(%%(G)PELLE (2,%%(VAR),%%(5+h*9)))
%%INCLUDE(PELLE (2,VAR,%%(5+h*9)))


  B = 13
  Hej svejs



Mary had a little lamb.
Peter had a little cat.
Mary had a little lamb
Peter had a little cat




1
1
1,2
A
1
{1,2}
1
2
{1,2}
{3,4}
LOCAL1                    V{rde p} P1 innan SET-satser
LOCAL2                    V{rde p} P2 innan SET-satser
LOCAL3                    V{rde p} P3 innan SET-satser
LOCAL4                    V{rde p} P4 innan SET-satser
                V{rde p} LOCAL1 innan SET-satser
                V{rde p} LOCAL2 innan SET-satser
                V{rde p} LOCAL3 innan SET-satser
                V{rde p} LOCAL4 innan SET-satser
LOCAL1                    V{rde p} P1 efter SET-satser
LOCAL2                    V{rde p} P2 efter SET-satser
LOCAL3                    V{rde p} P3 efter SET-satser
LOCAL4                    V{rde p} P4 efter SET-satser
Local1                V{rde p} LOCAL1 efter SET-satser
Local2                V{rde p} LOCAL2 efter SET-satser
Local3                V{rde p} LOCAL3 efter SET-satser
Local4                V{rde p} LOCAL4 efter SET-satser
%%(P1)                    V{rde p} P1 efter INCLUDE-satsen
%%(P2)                    V{rde p} P2 efter INCLUDE-satsen
%%(P3)                    V{rde p} P3 efter INCLUDE-satsen
%%(P4)                    V{rde p} P4 efter INCLUDE-satsen
Global1                V{rde p} LOCAL1 efter INCLUDE-satsen
Global2                V{rde p} LOCAL2 efter INCLUDE-satsen
Global3                V{rde p} LOCAL3 efter INCLUDE-satsen
Global4                V{rde p} LOCAL4 efter INCLUDE-satsen
Global1                    V{rde p} P1 innan SET-satser
Global2                    V{rde p} P2 innan SET-satser
Global3                    V{rde p} P3 innan SET-satser
Global4                    V{rde p} P4 innan SET-satser
                V{rde p} LOCAL1 innan SET-satser
                V{rde p} LOCAL2 innan SET-satser
                V{rde p} LOCAL3 innan SET-satser
                V{rde p} LOCAL4 innan SET-satser
Global1                    V{rde p} P1 efter SET-satser
Global2                    V{rde p} P2 efter SET-satser
Global3                    V{rde p} P3 efter SET-satser
Global4                    V{rde p} P4 efter SET-satser
Local1                V{rde p} LOCAL1 efter SET-satser
Local2                V{rde p} LOCAL2 efter SET-satser
Local3                V{rde p} LOCAL3 efter SET-satser
Local4                V{rde p} LOCAL4 efter SET-satser
%%(P1)                    V{rde p} P1 efter EINCLUDE-satsen
%%(P2)                    V{rde p} P2 efter EINCLUDE-satsen
%%(P3)                    V{rde p} P3 efter EINCLUDE-satsen
%%(P4)                    V{rde p} P4 efter EINCLUDE-satsen
Global1                V{rde p} LOCAL1 efter EINCLUDE-satsen
Global2                V{rde p} LOCAL2 efter EINCLUDE-satsen
Global3                V{rde p} LOCAL3 efter EINCLUDE-satsen
Global4                V{rde p} LOCAL4 efter EINCLUDE-satsen
Begin P1 = 0
R1[1] = 0

Local1 = 
Local1 = 0

Including with P1 = 1
 Begin P1 = 1
 R1[1] = 1

 Local1 = 
 Local1 = 1

 Including with P1 = 2
  Begin P1 = 2
  R1[1] = 2

  Local1 = 
  Local1 = 2

  Including with P1 = 3
   Begin P1 = 3
   R1[1] = 3

   Local1 = 
   Local1 = 3

   Including with P1 = 4
    Begin P1 = 4
    R1[1] = 4

    Local1 = 
    Local1 = 4

    Including with P1 = 5
     Begin P1 = 5
     R1[1] = 5

     Local1 = 
     Local1 = 5

     Including with P1 = 6
      Begin P1 = 6
      R1[1] = 6

      Local1 = 
      Local1 = 6

      Including with P1 = 7
       Begin P1 = 7
       R1[1] = 7

       Local1 = 
       Local1 = 7

       Including with P1 = 8
        Begin P1 = 8
        R1[1] = 8

        Local1 = 
        Local1 = 8

        Including with P1 = 9
         Begin P1 = 9
         R1[1] = 9

         Local1 = 
         Local1 = 9

         Including with P1 = 10
          Begin P1 = 10
          R1[1] = 10

          Local1 = 
          Local1 = 10


          Local1 = 10
          Local1 = 10

          R1[1] = 10
          End P1 = 10

         Local1 = 9
         Local1 = 9

         R1[1] = 10
         End P1 = 9

        Local1 = 8
        Local1 = 8

        R1[1] = 10
        End P1 = 8

       Local1 = 7
       Local1 = 7

       R1[1] = 10
       End P1 = 7

      Local1 = 6
      Local1 = 6

      R1[1] = 10
      End P1 = 6

     Local1 = 5
     Local1 = 5

     R1[1] = 10
     End P1 = 5

    Local1 = 4
    Local1 = 4

    R1[1] = 10
    End P1 = 4

   Local1 = 3
   Local1 = 3

   R1[1] = 10
   End P1 = 3

  Local1 = 2
  Local1 = 2

  R1[1] = 10
  End P1 = 2

 Local1 = 1
 Local1 = 1

 R1[1] = 10
 End P1 = 1

Local1 = 0
Local1 = 0

R1[1] = 10
End P1 = 0
